main/src/routes/trackRouter.js

```diff
@@ -1068,30 +1068,29 @@ let sampleData = {
 // output: returns html rendering of the tracks
 router.get('/', checkCredentials, function (req, res, next) {
     // use res.render("track", sampleData) for testing purposes
-
     if (req.app.locals.main.result) {
         return res.render("track", req.app.locals.main.result);
     } else {
-        // TODO: at this point, user's credentials are valid, but they do not have custom curated tracks
-        //  display an 'error' page, which tells user they need to take a photo, and has a button prompting to go back to homepage
-        return res.render('error', {
-            message: "No custom tracks yet, please take a photo to get curated tracks",
-            error: {status: 400, stack: "no stack"}
-        });
+        return res.redirect("/webcam/#/error/no-photo-submitted");
     }
 });
 
 // REQUIRES: req.body to contain a list of track URI's in format ["spotify:track:1ue7zm5TVVvmoQV8lK6K2H", ...]
 router.post('/', [trackLimiter, refreshCredentialsIfExpired], function (req, res, next) {
     let main = req.app.locals.main;
-    return main.createMoodPlaylist(req.body)
-        .then((playlistURL) => {
-            return res.status(200).json({link: playlistURL});
-        })
-        .catch((err) => {
-            console.log(err);
-            return res.status(400).json({"error": err.message});
-        })
+    if (main) {
```

Comment:
I don't think we need to check `main` here because the middleware `refreshCredentialsIfExpired` will check that